utils/zerovec/README.md

```diff
@@ -38,6 +38,14 @@ Benchmark results on x86_64:
 
 \* *This result is reported for `Vec<String>`. However, Serde also supports deserializing to `Vec<&str>`; this gives 1.8420 μs, much faster than `Vec<String>` but a bit slower than `zerovec`.*
 
+| Operation | `HashMap<K,V>`  | `LiteMap<K,V>` | `ZeroMap<K,V>` |
+|---|---|---|---|
+| Look up a `&str` key from a 16-element map | 46 ns | 41 ns | 38 ns |
+| Look up a `&str` key from a 1,048,576-element map | 49 ns | 211 ns | 171 ns |
+| Look up a `&str` key from a deserialized 1,048,576-element map | 51 ns | 220 ns | 269 ns |
+| Serialize a 16-element `<String, String>` map | 524 ns | 607 ns | 1.48 μs |
+| Deserialize a 16-element `<String, String>` map | 2.53 μs | 1.75 μs | 494 ns |
```

Comment:
Suggestion: The values I'd report would probably be

1. Deserialize a small map
2. Deserialize a large map
1. Lookup in a small deserialized map
2. Lookup in a large deserialized map

Reply:
Done.